docs/pages/admin-guides/teleport-policy/integrations/azure-sync.mdx

```diff
@@ -0,0 +1,125 @@
+
+- What resources are accessible to Azure users and roles?
+- Which groups belong to other groups and how does that impact resource access?
+- How broad or narrow is scoping across the subscription when assigning roles to users and groups?
+
+Utilizing the Access Graph to analyze permissions within an Azure subscription necessitates the setup of the Access
+Graph (AG) service, a Discovery Service, and integration with your Azure subscription.
+
+Access Graph is a feature of the [Teleport Policy](https://goteleport.com/platform/policy/) product that is available to Teleport Enterprise customers.
+
+After logging in to the Teleport UI, go to the Management tab. If enabled, Access Graph options can be found under
+the Permission Management section.
+
+## How it works
+
+Access Graph discovers Azure access patterns, synchronizes various Azure resources,
+including users, groups, service principals, and role definitions.
+These resources are then visualized using the graph representation detailed in the
+[Teleport Policy usage page](../policy-how-to-use.mdx).
+
+The importing process involves two primary steps:
+
+### Polling Cloud APIs
+
+The Teleport Discovery Service continuously scans the configured Azure subscriptions. At configurable intervals as
+low as 15 minutes, it retrieves the following resources from your Azure account:
+
+- Users
+- Groups
+- Service Principals
+- Role Definitions
+- Role Assignments
+- Virtual Machines
+
+Once all the necessary resources are fetched, the Teleport Discovery Service pushes them to the Access Graph, ensuring
+that it remains updated with the latest information from your Azure environment.
+
+### Importing resources
+
+Teleport Policy’s Access Graph feature delves into the resources retrieved from your Azure subscription, crafting a
+graphical representation thereof.
+
+## Prerequisites
+
+- A running Teleport Enterprise cluster v17.2.1 or later.
+- Teleport Policy enabled for your account.
+- For self-hosted clusters:
+- Ensure that an up-to-date `license.pem` is used in the Auth Service configuration.
+- A running Access Graph node v1.27.0 or later.
+Check the [Teleport Policy page](../teleport-policy.mdx) for details on
+how to set up Access Graph.
+- The node running the Access Graph service must be reachable from the Teleport Auth Service.
+
+## Step 1/2. Configure Discovery Service (Self-hosted only)
+
+<Notice type="warning">
+  If you have a managed Teleport Enterprise cluster, you can disregard
+  this step, as managed Teleport Enterprise already operates a properly configured
+  Discovery Service within your cluster.
+</Notice>
+
+To activate the Teleport Discovery Service, add the provided snippet to your Auth Service configuration.
+This service monitors dynamic `discovery_config` resources that are set up with the `discovery_group` matching
+<Var name="access-graph-disc" />.
+
+```yaml
+discovery_service:
+  enabled: true
+  discovery_group: <Var name="access-graph-disc" />
+```
```

Comment:
This is a little confusing, because we say to add this snippet to your auth service configuration, but the Teleport config has both a top level `auth_service` and `discovery_service` section.

Without more context, how can we expect the user to know where to insert this snippet?